extensions/azurePublish/src/components/azureProvisionDialog.tsx

```diff
@@ -211,6 +221,28 @@ const reviewCols: IColumn[] = [
   },
 ];
 
+const getHostname = (config) => {
+  if (config?.hostname) {
+    return config.hostname;
+  } else if (config?.name) {
+    return config?.environment ? `${config.name}-${config.environment}` : config.name;
+  }
+};
+
+const getDefaultFormData = (currentProfile, defaults) => {
```

Comment:
This really clarifies how the profile get overlaid on the extension state.  It was really confusing before.

Reply:
I was super confused as well.